content/blog/deploying-netlify-cms-on-aws/index.md

```diff
@@ -0,0 +1,363 @@
+    // ],
+
+    restrictions: {
+        geoRestriction: {
+            restrictionType: "none",
+        },
+    },
+
+    viewerCertificate: {
+        acmCertificateArn: certificateArn,  // Per AWS, ACM certificate must be in the us-east-1 region.
+        sslSupportMethod: "sni-only",
+    },
+
+    loggingConfig: {
+        bucket: logsBucket.bucketDomainName,
+        includeCookies: false,
+        prefix: `${config.targetDomain}/`,
+    },
+};
+
+const cdn = new aws.cloudfront.Distribution("cdn", distributionArgs);
+```
+
+### Step 5: Alias Record
+
+Next, we create an alias record that points to the CloudFront distribution.
+
+```typescript
+function createAliasRecord(
+    targetDomain: string, distribution: aws.cloudfront.Distribution): aws.route53.Record {
+    const domainParts = getDomainAndSubdomain(targetDomain);
+    const hostedZoneId = aws.route53.getZone({ name: domainParts.parentDomain }, { async: true }).then(zone => zone.zoneId);
+    return new aws.route53.Record(
+        targetDomain,
+        {
+            name: domainParts.subdomain,
+            zoneId: hostedZoneId,
+            type: "A",
+            aliases: [
+                {
+                    name: distribution.domainName,
+                    zoneId: distribution.hostedZoneId,
+                    evaluateTargetHealth: true,
+                },
+            ],
+        });
+}
+
+const aRecord = createAliasRecord(config.targetDomain, cdn);
+```
+
+---
+
+We've finished implementing our application infrastructure and we're ready to deploy. Run `pulumi up` to execute the Pulumi code to deploy to AWS. However, there is a more convenient way to do so with Github.
+
+## Github Workflow (Optional)
+
+[Github Actions](https://docs.github.com/en/actions) adopts the Continuous Delivery/Integration concept and specifies a series of steps to run and deploy the program. We can take advantage of it with our CMS.
+
+In the workflow, Github secrets are accessed from `${{ secrets.ACCESS_TOKEN }}` where `ACCESS_TOKEN` is the Github credential. We can specify the secret in the project setting of Github.
+
+![Github Secrets Settings](./github-secrets.jpg)
+
+`PULUMI_STACK` is the Pulumi stack name that we are using.
+`PULUMI_ACCESS_TOKEN` is required for the automatic process and can be generated by clicking the blue button "New Access Token" on the Settings of [Pulumi console](https://app.pulumi.com).
+
+![Pulumi Tokens](./pulumi-token.jpg)
+
+---
+
+## Summary and Next Step
+
+We have deployed the stand-alone React CMS application on AWS. The [complete code](https://github.com/pulumi/examples/tree/master/aws-ts-netlify-cms-and-oauth/cms) is included in [Pulumi's Example Repository](https://github.com/pulumi/examples).
+
+The next article will show how to deploy on AWS instead of Netlify. To do this, we describe how to substitute the Netlify Identity Service by writing an External OAuth Client-Server and deploying it on AWS. Stay tuned for our next blog post.
```

Comment:
```suggestion
The next article will show how to deploy on AWS instead of Netlify. We describe how to substitute the Netlify Identity Service by writing an External OAuth Client-Server and deploying it on AWS. Stay tuned for our next blog post.
```